packages/sync-actions/src/cart-discounts-actions.js

```diff
@@ -14,11 +14,12 @@ export const baseActionsList = [
   { action: 'changeStackingMode', key: 'stackingMode' },
 ]
 
-export function actionsMapBase(diff, oldObj, newObj) {
+export function actionsMapBase(diff, oldObj, newObj, config = {}) {
```

Comment:
Why not `config: SyncActionConfig = {}`?

Reply:
`flow` notation hasn't been added to this file at the time of writing.
The changes I've made where I added type notations where on files where `flow` was already added, and just needed "cleaning up" to eliminate all warnings.

I would want to say that we should add `flow` to all files but I think we should do that another time

Reply:
All cool. Thanks for clarifying.